broker-cli/commands/orderbook.js

```diff
@@ -17,37 +17,117 @@ const EVENT_TYPES = Object.freeze({
  * @param {string} market
- * @param {Array<Object>} asks - an object with price and depth
- * @param {Array<Object>} bids - an object with price and depth
+ * @param {Array<Object>} asks - an array of objects with price and depth
+ * @param {Array<Object>} bids - an array of object with price and depth
  */
 function createUI (market, asks, bids) {
   console.clear()
+  const windowHeight = size.get().height
+
+  // Fill as many orders as the screen allows less other info displayed
+  const maxLengthPerSide = Math.floor((windowHeight - 12) / 2)
   const baseCurrencySymbol = market.split('/')[0].toUpperCase()
-  const windowWidth = size.get().width
-  const { mainTableWidth, innerTableWidth } = calculateTableWidths(windowWidth)
-  const table = new Table({
-    head: ['ASKS', 'BIDS'],
-    style: { head: ['gray'] },
-    colWidths: [mainTableWidth, mainTableWidth]
+
+  const parentTable = new Table({
+    head: [],
+    chars: {
+      'top-mid': '',
+      'bottom-mid': '',
+      'left-mid': '',
+      'mid': '',
+      'mid-mid': '',
+      'right-mid': '',
+      'middle': ''
+    },
+    style: {
+      'padding-left': 0,
+      'padding-right': 0
+    }
   })
 
-  // The extensive options are required because the default for cli-table is to have
-  // borders between every row and column.
-  const innerTableOptions = {
-    head: ['Price', `Depth (${baseCurrencySymbol})`],
-    style: { head: ['gray'] },
-    colWidths: [innerTableWidth, innerTableWidth],
+  // Used for `chars` property when instantiating a new Table with no borders
+  const noBorders = {
+    'top': '',
+    'top-mid': '',
+    'top-left': '',
+    'top-right': '',
+    'bottom': '',
+    'bottom-mid': '',
+    'bottom-left': '',
+    'bottom-right': '',
+    'left': '',
+    'left-mid': '',
+    'mid': '',
+    'mid-mid': '',
+    'right': '',
+    'right-mid': '',
+    'middle': ''
+  }
+
+  const tableHeaders = new Table({
+    head: [],
+    chars: noBorders,
+    colWidths: [COLUMN_WIDTHS, COLUMN_WIDTHS],
+    style: {
+      'padding-left': 0,
+      'padding-right': 0
+    }
+  })
+
+  tableHeaders.push([{ hAlign: 'center', content: 'Price' }, { hAlign: 'center', content: `Depth (${baseCurrencySymbol})` }])
+
+  const asksTable = new Table({
+    head: [],
+    chars: noBorders,
+    colWidths: [COLUMN_WIDTHS, COLUMN_WIDTHS],
+    style: {
+      'padding-left': 0,
+      'padding-right': 0
+    }
+  })
+
+  const bidsTable = new Table({
+    head: [],
+    chars: noBorders,
+    colWidths: [COLUMN_WIDTHS, COLUMN_WIDTHS],
+    style: {
+      'padding-left': 0,
+      'padding-right': 0
+    }
+  })
+
+  // Used for creating border between bids / asks
+  const gapTable = new Table({
+    head: [],
+    rowHeights: [1],
+    // Only need bottom border, so remove all other characters
     chars: {
       'top': '',
       'top-mid': '',
       'top-left': '',
       'top-right': '',
-      'bottom': '',
```

Comment:
worthwhile to specify the character you want? I know it's in there by default, but might clarify this section